power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/serialization/deserializer.hpp

```diff
@@ -193,7 +193,7 @@ struct MapArrayVisitor : DefaultErrorVisitor<MapArrayVisitor<map_array>> {
     static constexpr bool enable_array =
         std::same_as<map_array, visit_array_t> || std::same_as<map_array, visit_map_array_t>;
     static constexpr std::string_view static_err_msg =
-        enable_map ? (enable_array ? "Expect a map or array." : "Expect a map.") : "Expect an array.";
+        enable_map ? (enable_array ? "Expected a map or array." : "Expected a map.") : "Expected an array.";
```

Comment:
Now sure this is good English. You could either do `A map or an array expected` or `Expecting a map or an array`.

Reply:
```suggestion
        enable_map ? (enable_array ? "Expected a map or an array." : "Expected a map.") : "Expected an array.";
```